gradle.properties

```diff
@@ -13,8 +13,8 @@ ballerinaGradlePluginVersion=0.14.1
 ballerinaLangVersion=2201.0.1
 
 #stdlib dependencies
-stdlibIoVersion=1.2.1
-stdlibRegexVersion=1.2.1
+stdlibIoVersion=1.2.2-20220407-142300-7060eb4
+stdlibRegexVersion=1.3.0-20220425-094000-d073680
```

Comment:
As discussed yesterday, when we are moving to minor versions of the dependent modules. we need to move our modules to minor versions.

Reply:
Updated module version to `2.3.0`